### ISSUE TITLE
Text Size

### ISSUE THREAD

**Issue by Annis-Monadjem** [egoist/devdocs-app#24]
Thanks for making and sharing `Devdoc`!
I have difficulty reading for long small text size of articles/reference pages and unfortunately, `Devdoc` comes with very small text size.
**Please make text size adjustable so that it can be customized by user.**

**Comment by egoist** [comment]
Checkout `v0.3.0` release: https://github.com/egoist/devdocs-app/releases/tag/v0.3.0

You can add following CSS to your custom CSS file:

```css
._list-item, ._content {
  font-size: 1rem !important;
}
```

**Comment by Annis-Monadjem** [comment]
@egoist: Subject: to custom increase content font. I followed your instructions for `custom.css` and it is not working for me. Any idea's why? I have upgraded to v0.5.1 and added to `custom.css`:
> ._list-item, ._content {
  font-size: 1rem !important;
}
Next, I tweeted different sizes 1rem, 5rem, 10rem - but none increase the size of font content. The small font size that is used by default in devdocs.io is prohibitive so effectively i end up not using devdocs at all. It would be really nice to have this feature work.
